pom.xml

```diff
@@ -93,7 +93,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>2.1.16</version>
+            <version>2.2.0</version>
```

Comment:
The credentials configurator moved to the credentials plugin from the support plugin in this version